README.md

```diff
@@ -1,5 +1,130 @@
-# Hackathon
+# Global Legal Hackathon
 
 <br>
 
+### Project Contacts
+
+Matthew Schneider
+    [INSERT]
+
+Jordan L. Fischer, Esq.
+    Teaching Professor of Law
+    Thomas R. Kline School of Law at Drexel University
+    [jlf324@drexel.edu](jlf324@drexel.edu)
+
+Cameron Bale
+    [INSERT]
+
```

Comment:
PhD Student in Business Analytics
LeBow College of Business at Drexel University